bok_choy/browser.py

```diff
@@ -204,8 +205,13 @@ def browser_check_func():
             LOGGER.debug('Failed to instantiate browser: ' + err.strerror)
             return False, None
 
+        except WebDriverException as err:
+            msg = str(err)
+            LOGGER.debug(msg)
+            return False, None
+
     browser_instance = Promise(
-        browser_check_func, "Browser is instantiated successfully.", timeout=30).fulfill()
+        browser_check_func, "Browser is instantiated successfully.", try_limit=5).fulfill()
```

Comment:
Is it reasonable that this would fail 4 times but succeed the 5th? I'm thinking we might want to set the try_limit lower, maybe 2 or 3.